README.md

```diff
@@ -15,3 +44,66 @@ table = Table("table-id", MetaData(bind=engine), autoload=True)
 for row in select(["*"], from_obj=table).execute().fetchall():
     print(row)
 ```
+
+**Create a table**
+```python
+from sqlalchemy import (
+    Column,
+    Integer,
+    MetaData,
+    String,
+    Table,
+    create_engine,
+)
+
+engine = create_engine(
+    "spanner:///projects/appdev-soda-spanner-staging/instances/sqlalchemy-dialect-test/databases/compliance-test"
+)
+metadata = MetaData()
+
+user = Table(
+    "users",
+    metadata,
+    Column("user_id", Integer, primary_key=True),
+    Column("user_name", String(16), nullable=False),
+)
+
+metadata.create_all(engine)
+```
+Features and limitations
+-----------
+**Unique constraints**  
+Cloud Spanner doesn't support direct UNIQUE constraints creation. In order to achieve column values uniqueness UNIQUE indexes should be used.
+
+Instead of direct UNIQUE constraint creation:
+```python
+Table(
+    'table',
+    metadata,
+    Column('col1', Integer),
+    UniqueConstraint('col1', name='uix_1')
+)
+```
+Create a UNIQUE index:
+```python
+Table(
+    'table',
+    metadata,
+    Column('col1', Integer),
+    Index("uix_1", "col1", unique=True),
+)
+```
+**Autocommit mode**  
+Spanner dialect supports both "autocommit" and "manual commit" modes. To set/change the current mode isolation levels can be used: `SERIALIZABLE` and `AUTOCOMMIT`.
+
+**DDL and transactions**  
+DDL statements are executed aside of regular transactions mechanism, which means DDL statements will not be rolled back on normal transaction rollback.
+
+**Dropping a table**  
+Cloud Spanner, by default, doesn't drop tables, which has secondary indexes and/or foreign key constraints. In Spanner dialect for SQLAlchemy, however, this restriction is omitted - if a table you are trying to delete has indexes/foreign keys, they will be dropped automatically right before dropping the table.
+
+**Other limitations**  
+- WITH RECURSIVE statement is not supported.
+- Named schemas are not supported.
+- Temporary tables are not supported, real tables are used instead.
+- Numeric type dimensions (scale and precision) are constant. See the [docs](https://cloud.google.com/spanner/docs/data-types#numeric_types).
```

Comment:
Some other sections to include either here or in a separate file:
- Type mapping: I think people would like to have a reference for this in the README instead of having to look through the code.
- Information or any pointers on how to migrate an existing database into CSpanner.
- Running the tests against prod and/or the emulator: this can also go in CONTRIBUTING.md.
- License link.

Reply:
Done